apps/web/components/NavTabs.tsx

```diff
@@ -62,7 +62,7 @@ const NavTabs: FC<NavTabProps> = ({ tabs, linkProps, ...props }) => {
           const className = tab.className || "";
           return (
             <Component key={tab.name}>
-              <Link key={tab.name} href={href} {...linkProps}>
+              <Link key={tab.name} href={href} {...linkProps} legacyBehavior>
```

Comment:
do we want to use legacyBehavior or just refactor and remove the `<a>`?

Reply:
Depends on the use case